.vsts-ci/phase.yml

```diff
@@ -45,9 +50,9 @@ jobs:
           }
         }
         Write-Host "Building snap..."
-        ./tools/releaseBuild/vstsbuild.ps1 -Name $buildName
+        snapcraft --destructive-mode
```

Comment:
can you add a comment what this does

Reply:
```suggestion
        # destructive-mode leaves files and packages on the machine 
        # This is okay because Azure DevOps Hosted machines are ephemeral.
        snapcraft --destructive-mode
```